attribute-projection-registry/build.gradle.kts

```diff
@@ -1,15 +1,42 @@
 plugins {
   `java-library`
   jacoco
-  id("org.hypertrace.jacoco-report-plugin")
-  id("org.hypertrace.publish-plugin")
+  alias(commonLibs.plugins.hypertrace.jacoco)
+  alias(commonLibs.plugins.hypertrace.publish)
+}
+
+configurations {
```

Comment:
from here to line 32 would likely go into a plugin, which applies something similar to all java projects.